timetable/templates/index.html

```diff
@@ -275,7 +275,7 @@ <h3>Select your school to begin</h3>
 						</a></li>
 				</ul>
 			</div>
-			<h6>COPYRIGHT © 2017-2022 SEMESTERLY TECHNOLOGIES, LLC.</h6>
+			<h6>COPYRIGHT © 2017 SEMESTERLY TECHNOLOGIES, LLC.</h6>
```

Comment:
Do you know why it's deleting the -2022 edits

Reply:
Not sure, I merged Sebastian's change so I think he changed this file?

Reply:
#657

Reply:
huh, okay